### PR TITLE
Update .zappr.yaml

### PR DESCRIPTION
**Pull request by prayerslayer**
we still have to do stuff
- [x] done task
- [ ] open task
like for real

### CODE REVIEW

**Comment by prayerslayer** [comment]
Oh im sorry

**Comment by prayerslayer** [comment]
what is going on here

**Comment by prayerslayer** [comment]
👍

**Comment by prayerslayer** [comment]
👍

**Comment by prayerslayer** [comment]
:-1:

**Comment by prayerslayer** [comment]
this comment is unrelated

**Comment by prayerslayer** [comment]
👍

**Comment by prayerslayer** [comment]
👍

File: .zappr.yaml [hunk 1]
```diff
@@ -1,7 +1,7 @@
-# this is how you do stuff with zappr
+# this is how you configure zappr
 # a more exhaustive description is at
 # https://zappr.readthedocs.io/en/latest/setup
-
+# unreleated change
```
Comment (zappr-test) [inline .zappr.yaml]: What are you doing

**Review by zappr-test — DISMISSED** [review]
this is unacceptable

File: .zappr.yaml [hunk 2]
```diff
@@ -22,3 +22,8 @@
 #   (you can also specify `orgs` and `users` and define `groups`)
 #   from:
 #     collaborators: true
+
```
Comment (zappr-test) [inline .zappr.yaml:25]: THIS IS WHAT GRINDS MY GEARS

**Review by zappr-test — DISMISSED** [review]
Inacceptable

**Comment by prayerslayer** [comment]
👍

**Comment by prayerslayer** [comment]
👍

**Comment by prayerslayer** [comment]
👍

**Comment by prayerslayer** [comment]
👍

**Comment by prayerslayer** [comment]
👍

**Comment by prayerslayer** [comment]
👍

File: .zappr.yaml (hunk 1 above)
Comment (zappr-test) [reply .zappr.yaml]: Can you fix this typo real quick

File: .zappr.yaml (hunk 2 above)
Comment (zappr-test) [reply .zappr.yaml:25]: What is this shit

**Review by zappr-test — DISMISSED** [review]
I cannot do this for you

File: .zappr.yaml (hunk 1 above)
Comment (zappr-test) [reply .zappr.yaml]: You cant be serious

**Review by zappr-test — CHANGES_REQUESTED** [review]
Maybe you can do better

**Review by zappr-test — APPROVED** [review]

File: .zappr.yaml (hunk 1 above)
Comment (zappr-test) [reply .zappr.yaml]: starting a review

**Review by zappr-test — CHANGES_REQUESTED** [review]
no

**Review by prayerslayer — COMMENTED** [review]
asdf

**Pull request closed without merging**